src/Components/Cluster/Cluster.spec.ct.js

```diff
@@ -62,7 +63,10 @@ describe('cluster page', () => {
     // renders table component
     cy.get(RULES_TABLE).should('have.length', 1);
     // test how many rows were rendered
-    checkRowCounts(singleClusterPageReport.report.data.length);
+    // TODO: for selectable rows, tbody is rendered for each row, and checkRowCounts doesn't work correctly
```

Comment:
maybe then we should improve the checkRowCounts function rather than creating a ad-hoc test